recipes/r-kdetrees/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set version = '0.1.5' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-kdetrees
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:     {{ cran_mirror }}/src/contrib/Archive/kdetrees/kdetrees_{{ version }}.tar.gz
+  sha256: af65c382b3e52a2bfe7ed8b9a3ff05f2cdd0a1a122c972ed2f79d02dae178ede
+
+build:
+  skip: true  # [win]
```

Please switch back to noarch, as the packages has not compiled code and will be build on linux only after the feedstock conversion. So win CI failures can be ignored here.
```suggestion
  noarch: generic
```